src/wizard.ts

```diff
@@ -0,0 +1,83 @@
+import * as fs from 'fs'
+import { prompt, PromptObject } from 'prompts'
+
+interface ProjectOptions {
+  projectName: string
+  documentSyntax: string
+  entryPoint: string
+}
+
+export const getProjectNamePrompt = (
+  processDirectory: string,
+): PromptObject => ({
+  type: 'text',
+  name: 'projectName',
+  message: 'Project name',
+  initial: processDirectory,
+  validate: (input) =>
+    !!input.match(/^[\w\-_]+$/) ||
+    'Project name may only contain letters, hyphens and underscores',
+})
+
+export const getDocumentSyntaxPrompt = (): PromptObject => ({
+  name: 'documentSyntax',
+  type: 'select',
+  message: 'Document syntax',
+  choices: [
+    { title: 'json', value: 'json' },
```

Comment:
Just an idea to use the proper capitalisation in the human-readable labels.
```suggestion
    { title: 'JSON', value: 'json' },
    { title: 'YAML', value: 'yaml' },
```